.github/workflows/build_and_test.yml

```diff
@@ -1049,6 +1012,7 @@ jobs:
         run: rm -rf ~/.ecr
 
   trigger-custom-extensions-build-and-wait:
+    if: github.event.action != 'ready_for_review'
```

Comment:
And here 😄

Reply:
same like before this one reruns when we mark PR ready for review

Reply:
removed